pom.xml

```diff
@@ -9,9 +9,9 @@
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>com.example</groupId>
-	<artifactId>weather.client</artifactId>
+	<artifactId>weather.com.example.weather.client.client</artifactId>
```

Comment:
Extremely odd artifact name... 
Usually groupId is page in reverse order, so `com.example` like here and artifactId is the name of project, so suitable would be `weather-client`.